fuse.js

```diff
@@ -11,7 +11,11 @@ storybookManager.bundle("> index.js")
 const fuse = fsbx.FuseBox.init({
   homeDir: "src",
   tsConfig: "./tsconfig.json",
-  outFile: "./assets/reaction-force.js",
+  outFile: "./reaction-force.js",
+    sourceMap: {
+      bundleReference: "./reaction-force.js.map",
+      outFile: "./reaction-force.js.map",
+    },
```

Comment:
ah yeah, was meant to make a note, this was due to all of the relative paths inside the source mapping setup, I couldn't find a way to say that they are relative to to `../`of the assets dir instead of `.`. 

Can give it another look at some point too. Might be something I've missed now there's a working version to compare against.